src/cumulus/commands/init.py

```diff
@@ -13,49 +13,53 @@ def init_docker_compose(self):
             'version': DOCKER_COMPOSE_VERSION,
             'services': {}
         }
-
+        project = "example"
+        hostname = project+".cumulus.com"
         self.already_started = []
         new_services = self.options['<service>']
         clean = self.options['--clean']
         for service in new_services:
             if(os.path.exists('./'+service) and not clean):
                 self.already_started.append(service)
                 continue
-                
+            name = project+"_"+service
             if service in WEB_APP:
                 doc['services'][service] = {
-                    'image': DOCKER_HUB + service,
+                    # 'image': DOCKER_HUB + service, uncomment before merging
```

Comment:
"uncomment before merging" ?

Reply:
If anyone wanted to try it out, I have changed the docker image so it won't be compatible with current docker image online.